.github/workflows/production-heartbeat.yml

```diff
@@ -11,28 +11,148 @@ on:
 jobs:
   production-heartbeat:
     strategy:
-      # By default, if any job in a matrix fails, all other jobs are immediately cancelled. This makes the jobs run to completion instead.
       fail-fast: false
       matrix:
-        os: [{vm: ubuntu-latest, exe: .sh}, {vm: windows-2019, exe: .cmd}]
+        os: [{vm: ubuntu-latest, exe: .sh}, {vm: windows-2019, exe: .cmd}, {vm: macos-latest, exe: .sh}]
```

Comment:
Where is exe field being used? 
I see we are using the vm field... but I don't see the exe field being used anywhere.

Reply:
Fixed.